src/scss/blocks/page.scss

```
@@ -0,0 +1,34 @@
+.page {
+  font-family: $font-family;
+  background-color: $bgr-color;
+  color: $main-color;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
```

Можно написать просто 
```suggestion
  text-size-adjust: 100%;
```

а автопрефиксер (который мы ыключили в вебпаке) уже сам создаст правила с префиксами для поддерживаемых браузеров.